blueflood-http/src/main/java/com/rackspacecloud/blueflood/inputs/handlers/HttpMetricsIngestionHandler.java

```diff
@@ -99,19 +99,32 @@ public void handle(ChannelHandlerContext ctx, HttpRequest request) {
 
             requestCount.inc();
             final String tenantId = request.getHeader("tenantId");
-            JSONMetricsContainer jsonMetricsContainer = null;
+            JSONMetricsContainer jsonMetricsContainer;
+            List<Metric> metrics;
+
             final Timer.Context jsonTimerContext = jsonTimer.time();
 
             final String body = request.getContent().toString(Constants.DEFAULT_CHARSET);
             try {
                 jsonMetricsContainer = createContainer(body, tenantId);
 
                 if (jsonMetricsContainer == null) {
-                    log.warn(ctx.getChannel().getRemoteAddress() + " No valid metrics");
+                    log.warn(ctx.getChannel().getRemoteAddress() + " No valid json content.");
                     DefaultHandler.sendResponse(ctx, request, "No valid metrics", HttpResponseStatus.BAD_REQUEST);
```

Comment:
It was to distinguish between the two errors in the log and to make the first log reflect more accurately why it was an error, but i didn't want to change the response message and keep all responses the same for now.  The else handles a new state we didn't have before here because we now process json on container creation and not with a separate method call later, so we check for empty json metrics here.

I changed both logs now to reflect the errors it much better.